packages/agoric-cli/src/sdk-package-names.js

```diff
@@ -5,6 +5,7 @@ export default [
   "@agoric/async-flow",
   "@agoric/base-zone",
   "@agoric/benchmark",
+  "@agoric/benchmark-test",
```

Comment:
This was automatically generated from the change to the packages's package.json `name:`. This was one of the impediments to doing a `yarn build` at the root directory.